final desc e initial influence measures (#14)
AlissonRP/Pokemom: relat.Rmd

```
Assim, como esperado os pokémom lendários possuem atributos superiores (na média) do que os não lendários, note que a força do ataque especial dos Pokémon lendários é `r round(mean_leg[2,5] / mean_leg[1,5],2)` vezes maior que os não lendários.  Para conjecturar a cerca dos atributos dos Pokémon, podemos averiguar os valores pormenorizados na Tabela \@ref(tab:summary).

```{r summary}
df |>
  mypdf1::pdf1_summary() |> rename(média = mean, mediana = median, `desvio padrão` = sd)|> mypdf1::pdf1_tbl("Medidas para resumir as variáveis númericas")
```

> Na Tabela \@ref(tab:summary) tem-se medidas para a média, mediana, desvio padrão, mínimo e máximo, as medidas para média e mediana são consideralvemente próximas, em torno de 70, com apenas a variável Attack com média `r mean(df$Attack)`. Conjecturando sobre o desvio padrão é plausível afirmar que que as médias para o Attack e Sp. Atk são as menos informativas, dado que, as amplitudes são menores que HP, Defense e Sp. Def, são os atributos que possuem maior desvio padrão. No atributo HP tem-se uma conjuntura peculiar, o fato de seu mínimo ser `r min(df$HP)`, o que dá-se pelo Pokémon Shedinja, quem tem um conjunto de habilidades especiais, que pode ser lido [\textcolor{blue}{aqui}](https://www.einerd.com.br/quem-e-shedinja-o-misterioso-pokemon-que-e-quase-invencivel/).

Reviewer: ```suggestion
Na Tabela \@ref(tab:summary) tem-se os valores da média, mediana, desvio padrão, mínimo e máximo, as medidas para média e mediana são consideralvemente próximas, em torno de 70, com apenas a variável Attack com média `r mean(df$Attack)`. Conjecturando sobre o desvio padrão é plausível afirmar que que as médias para o Attack e Sp. Atk são as menos informativas, dado que, as amplitudes são menores que HP, Defense e Sp. Def, são os atributos que possuem maior desvio padrão. No atributo HP tem-se uma conjuntura peculiar, o fato de seu mínimo ser `r min(df$HP)`, o que dá-se pelo Pokémon Shedinja, quem tem um conjunto de habilidades especiais,  que podem ser lidas [\textcolor{blue}{aqui}](https://www.einerd.com.br/quem-e-shedinja-o-misterioso-pokemon-que-e-quase-invencivel/).
```

# Análise Inferencial

> Nessa seção verificaremos uma etapa importante na análise de um ajuste de um modelo de regressão, a análise inferencial, em que buscar encontrar possíveis distorções das suposições do modelo, principalmente deturpações no componente aleatório e observações discrepantes, juntamente com análise de resíduos, adequação da distribuição proposta, assim então validando o modelo. Ademais a análise inferencial também se ocupará de testes de hipóteses, apresentação, seleção do modelo e verificação das variáveis que mais influenciam a predição sendo negativa ou positivamente, determinando se a perspectiva tem sentido prático.  

Reviewer: O *componente aleatório* não foi citado em lugar algum
```suggestion
Nessa seção verificaremos uma etapa importante na análise de um ajuste de um modelo de regressão, a análise inferencial, em que busca encontrar possíveis distorções das suposições do modelo, principalmente deturpações no componente aleatório e observações discrepantes, juntamente com análise de resíduos, adequação da distribuição proposta, assim então validando o modelo. Ademais a análise inferencial também se ocupará de testes de hipóteses, apresentação, seleção do modelo e verificação das variáveis que mais influenciam a predição sendo negativa ou positivamente, determinando se a perspectiva tem sentido prático.  
```

## Análise de Dignóstico

> Percebe-se que a análise e detecção de pontos influentes é um tópico relevante para a avaliação e validação de um modelo assim, nessa seção iremos avaliar a existência de observações atípicas, isto é, pontos que exercem peso desproporcional nas estimativas do modelo de Regressão Logística, por isso sucederemos com análise de pontos de avalanca, distância de cook, dffits e envelope simulado.

Reviewer: ```suggestion
Percebe-se que a análise e detecção de pontos influentes é um tópico relevante para a avaliação e validação de um modelo, assim nessa subseção iremos avaliar a existência de observações atípicas, isto é, pontos que exercem peso desproporcional nas estimativas dos parâmetros do modelo de Regressão Logística, por isso sucederemos com análise de pontos de avalanca, distância de cook, dffits e envelope simulado.
```

```

### Alavancagem
Nessa seção veremos as medidas de alavancagem, que informam se uma observação é discrepante em termos de covariável, ou seja, utilizando os resíduos busca medir a discrepância entre o valor observado e o valor ajustado, então na Figura \@ref(fig:d1), temos os valores da medida de alavancagem para cada observação.

```{r d1, fig.cap="Medidas de Alavancagem para o Modelo 1"}
alavanca(fit1,df)+ ggrepel::geom_text_repel(aes(label=1:nrow(df)), size=2.8, point.padding = 0.3)
```

Com a Figura \@ref(fig:d1), tem-se que a observação que possui a maior alavancagem é o ponto 262 (`r nome[262,1]`), devido ao fato que este Pokémon é o que mais achata a Figura, mas também temos outras observações acima do limite da medida de alavancagem estipulado.

### Distância de Cook

> Outra medida interessante para ponto aberrantes é a Distância de Cook ($D_i$), que mede essencialmente a influência das observações sobre os parâmetros e o ajuste, avaliando a influência de o que pequenas pertubarções nas variâncias das observações causam nas estimativas dos parâmetros. Ou de forma simplificada, temos a influência da observação $i$ sobre todos os $n$ valores ajustados, 

Reviewer: pra que escrever ($D_i$), não foi usado isso em lugar algum

```{r d2, fig.cap="Distância de Cook para o Modelo 1"}
cook(fit1, "Distância do Modelo 1", df) + ggrepel::geom_text_repel(aes(label=1:nrow(df)), size=2.8, point.padding = 0.3)
```

> Com a Figura \@ref(fig:d2), tem-se que as observações que possuem maior valores da distância de cook são: 72,262,414,415,416 e 430 (`r paste(nome[c(72,262,414,415,416,430),1], sep=' ')`, respectivamente), devido ao fato que estes Pokémons são os que estão mais distantes do intevalo definido, no entanto também existências outras observações fora do limite.

Reviewer: ```suggestion
Com a Figura \@ref(fig:d2), tem-se que as observações que possuem maior valores da distância de cook são: 72, 262, 414, 415, 416 e 430 (sendo os Pokémons conhecidos como: `r paste(nome[c(72,262,414,415,416,430),1], sep=' ')`, respectivamente), devido ao fato que estes Pokémons são os que estão mais distantes do intevalo definido, no entanto também existem outras observações fora dos limites.
```

### DFFITS

> A medida DFFITS ($DFFITS_i$) pode ser considerada uma medida complementar ou concorrente a distância de Cook, tendo o próposito de medir a influência das observações nos parâmetros de posição e escala (mas não simultaneamente), assim informam o grau de influência que a observação $i$ tem sobre o valor seu próprio valor ajustado $\hat{y_i}$, tem-se que:

Reviewer: ($DFFITS_i$) mesmo comentário do ($D_i$), cuidado que aqui estamos predizendo $\hat{p_i}$

```{r d3, fig.cap="DFFITS do Modelo 1"}
dffts(fit1, "DFFITS do Modelo 1", df)+ ggrepel::geom_text_repel(aes(label=1:nrow(df)), size=2.8, point.padding = 0.3)
```

> Com a Figura \@ref(fig:d3) percebe-se que as observações que podem ser consideradas pontos influentes de acordo com o DFFITS são: 262, 415, 416, 429 e 430, `r paste(nome[c(262,415,416,429,430),1],  sep=' ')`, respectivamente), visto que são os indíviduos que mais distorcem a Figura \@ref(fig:d3).

Reviewer: Coloca aqui também falando que
```r
paste(nome[c(262,415,416,429,430),1],  sep=' ')`
```

é o nome dos pokémon

### Resíduos

> A análise específica de resíduos também é importante para verificarmos visualmente a média dos resíduos e se existe algum valor fora do limite de 3 desvios padrões, pois esses possui baixíssima probabilidade de serem observados, então utilizando o resíduo deviance as suas ocorrências na Figura \@ref(fig:d4).

Reviewer: Pq vc quer verificar a **média** do resíduo aqui?
```suggestion
A análise específica de resíduos também é importante para verificarmos visualmente a média dos resíduos e se existe algum valor fora do limite de 3 desvios padrões, pois esses possui baixíssima probabilidade de serem observados, então utilizando o resíduo deviance as suas ocorrências podem ser observadas na Figura \@ref(fig:d4).
```

```{r d4, fig.cap="Resíduos Deviance do Modelo 1"}
residuals(fit1,type="deviance")|>
Resid("Resíduos do Modelo 1", df)
```
Com a Figura \@ref(fig:d4) verificamos que todos as ocorrências estão dentro dos limites: 

### Envelope Simulado

Finalizamos a análise de influência com o envolope simulado que permite uma melhor comparação entre os resíduos e os percentis da distribuição, fornecendo um vislumbre se a distribuição é adequada para o ajuste, como percebemos na Figura \@ref(fig:d5).

```{r, results = F, fig.show='hide'}
g1 <- hnp::hnp(fit1, resid.type="deviance", halfnormal = F)
```

```{r d5, fig.cap="Envelope Simulado dos dados com a Distribuição Binomial"}
geom_hnp(g1)
```

> Analisando a Figura \@ref(fig:d5), pode-se identificar que existem pontos fora das bandas simuladas, fornecendo indícios de que existam pontos influentes ou que a distribuição não é adequada para o ajuste.

Reviewer: ```suggestion
Analisando a Figura \@ref(fig:d5), pode-se identificar que existem  alguns pontos fora das bandas simuladas, fornecendo indícios de que existem pontos influentes ou que talvez  a distribuição não é adequada para o ajuste.
```
